### ISSUE TITLE
Missing heading menus

### ISSUE THREAD

**Issue by LogicalUnit** [t94xr/PCem_Linux#7]
This is fantastic -- I've been trying all day to get PCem working on my Ubuntu setup all day. Unfortunately, the header menus appear to be missing, and I can't change CD-ROMs to install Windows (95/98). Any idea what to do about this?

![Screenshot from 2024-04-27 18-19-41](https://github.com/t94xr/PCem_Linux/assets/6601496/85420416-4dee-4b51-9e72-085ccf8cb7fd)
![Screenshot from 2024-04-27 18-19-51](https://github.com/t94xr/PCem_Linux/assets/6601496/069dae1e-10a4-4178-9690-e1806a2ba3a6)

**Comment by t94xr** [comment]
I'm guessing since PCem17 is an X11 program, you will experience issues on a Wayland-only OS such as the latest version of Ubuntu - that you're indeed using right?

**Comment by LogicalUnit** [comment]
Yep, I'm on Jammy Jellyfish 24.04 LTS. I'm still newish to Linux and learning things. Is there any workaround for this?

**Comment by t94xr** [comment]
No sadly, PCem will need a massive update to support Wayland.
It's funny that a VR dev is "newish to Linux" ... But its a great OS, especially as a home dev server. 

But it would sound like that PCem may be getting updated, from the looks of it, [almost completely overhauled](https://pcem-emulator.co.uk/phpBB3/viewtopic.php?t=3799).

I will edit the repo to reflect that this will age out **_very_** quickly as a resource now that Wayland is being adopted.
